### DISCUSSION TITLE
How you extended Reqnroll Quickstart sample app

### DISCUSSION THREAD

**Discussion by gasparnagy** [reqnroll/Reqnroll#6] [Show and tell]
Post your solutions and ideas on how you extended the [Quickstart tutorial](https://docs.reqnroll.net/latest/quickstart/index.html) application.

**Comment by ZsoltDunai** [comment]
Hey hey,
@gasparnagy  its good to see SF went forward with a new name. 

Well I did not extend the sample, but I created a simple Demo project for Reqnroll with PlayWright. If this could be interesting to anyone I leave here the link: https://github.com/ZsoltDunai/ReqnrollTestProjectWithPlayWright

**Reply by gasparnagy** [reply]
Oh, this is super cool. I quickly created a ["Samples" page](https://docs.reqnroll.net/latest/help/samples.html) in our docs. Could you list your sample there as well? https://github.com/reqnroll/Reqnroll/blob/main/docs/help/samples.md

Also I encourage you to write a small blog post about it!

**Comment by MalcolmCrowe** [comment]
Hi, All this is great but I can't seem to use debug things.
I am using Visual Studio 2022, with .Net 6.0. If I set a breakpoint in the feature file, the test explorer simply ignores the break point. If I use Start Debugging instead, it seem to hesitate at the breakpoint and briefly shows a debugger window (maybe Threads) but the program exits. I have checked Debug in the Build project properties, and unchecked Just My Code in the Debugger Options. Any ideas?

**Reply by gasparnagy** [reply]
@MalcolmCrowe Did you use the "Debug" command from the context menu of the test explorer to debug the tests?

![image](https://github.com/reqnroll/Reqnroll/assets/144701/3b1682df-8384-4222-bae2-99748a5d5072)

(The "Start Debugging" runs your application, not the tests, so that behavior is OK.)
